app/lib/test_pals_mock_ldap.rb

```diff
@@ -0,0 +1,31 @@
+class TestPalsMockLDAP
+
+  def initialize(options)
+    @options = options
+    @email = options[:auth][:username]
+    @password = options[:auth][:password]
+  end
+
+  def bind
```

Comment:
Should we simulate the worst-case scenario for how an LDAP server might be configured here?  That way tests can validate that we're defensive for this in our code.

Reply:
Good idea 😄

Reply:
Implemented this here: e871e596c6355128824da587cf9c91759bbd568b.